### PR TITLE
Update Crashlytics SDK to use new Android API for build version

### PR DESCRIPTION
**Pull request by samedson**

### CODE REVIEW

File: firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/internal/common/AppData.java [hunk 1]
```diff
@@ -46,7 +47,7 @@ public static AppData create(
     final String installerPackageName = idManager.getInstallerPackageName();
     final PackageManager packageManager = context.getPackageManager();
     final PackageInfo packageInfo = packageManager.getPackageInfo(packageName, 0);
-    final String versionCode = Integer.toString(packageInfo.versionCode);
+    final String versionCode = AppData.getAppBuildVersion(packageInfo);
```
Comment (mrober) [inline firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/internal/common/AppData.java:50]: Do we need the `AppData.` part?

**Review by mrober — APPROVED** [review]

File: firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/internal/common/AppData.java (hunk 1 above)
Comment (samedson) [reply firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/internal/common/AppData.java]: Removed

File: firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/internal/common/AppData.java [hunk 2]
```diff
@@ -61,6 +62,14 @@ public static AppData create(
         developmentPlatformProvider);
   }
 
+  private static String getAppBuildVersion(PackageInfo packageInfo) {
+    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.P) {
+      return Long.toString(packageInfo.getLongVersionCode());
+    } else {
+      return Integer.toString(packageInfo.versionCode);
```
Comment (visumickey) [inline firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/internal/common/AppData.java:69]: Should we suppress the deprecation message? 

`@Suppress("DEPRECATION") packageInfo.versionCode`

**Review by visumickey — APPROVED** [review]

File: firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/internal/common/AppData.java (hunk 2 above)
Comment (samedson) [reply firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/internal/common/AppData.java:70]: 👍

**Pull request merged**